static/assets/geojson/herrenberg-old-town.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This is exactly what I expected. GeoJSON is just JSON that follows a formal specification.

Reply:
:+1:

Reply:
However, if the file ends in `.geojson` then Github will automatically display it on a map. This is not needed here though.